src/geometry/translation.rs

```diff
@@ -15,23 +15,18 @@ use crate::base::{Const, DefaultAllocator, OMatrix, SVector, Scalar};
 
 use crate::geometry::Point;
 
-#[cfg(feature = "rkyv-serialize")]
-use rkyv::bytecheck;
-
 /// A translation.
 #[repr(C)]
 #[cfg_attr(
     feature = "rkyv-serialize-no-std",
     derive(rkyv::Archive, rkyv::Serialize, rkyv::Deserialize),
-    archive(
-        as = "Translation<T::Archived, D>",
```

Comment:
This `as` being removed is a UX downgrade because we can't access underlying functions from original type on rkyv archived type.